main.cpp

```diff
@@ -1,13 +1,49 @@
 #include "mbed.h"
```

Comment:
Please remove all test code from main. Include only code that will be used in the car (initialize the analog and digital polling outside the while loop, then have the while loop do nothing)